confidential-data-hub/Makefile

```diff
@@ -19,8 +19,8 @@ SOURCE_ARCH := $(shell uname -m)
 ARCH ?= $(shell uname -m)
 DEBUG ?=
 LIBC ?= gnu
-RESOURCE_PROVIDER ?=
-PROVIDER ?=
+RESOURCE_PROVIDER ?= kbs,sev
```

Comment:
This is awaward for Makefile. If we does not want any `RESOURCE_PROVIDER`
```shell
make RESOURCE_PROVIDER=
```
will still set `RESOURCE_PROVIDER` to `kbs,sev`, because in Makefile an empty-value variable is equal to no-this-variable


The case is reasonable, because `kbs` feature only enables the ability of connecting to CoCoKBS. If we only want use legacy `offline-fs-kbc`, we does not need any features for `RESOURCE_PROVIDER`

Reply:
Are you ok with the default RESOURCE_PROVIDER set to "kbs,sev" ?
Or you prefer a different default ?

Reply:
It sounds good